spec/fixtures/forms/2021_2022.json

```diff
@@ -31,7 +31,7 @@
               "person_1_age": {
```

Comment:
We don't care too much about the wording in our test fixtures. The idea behind it being, if we change content for the actual form that doesn't impact the tests. It doesn't hurt having this as a change, but for the next time, it's not necessary

Reply:
Yeah, I’m always unsure how important/useful it is to keep the fixtures and tests up to date with the actual form. OCD usually gets the better of me! 🤪